joomla/joomla-cms
# Moving "afterDisplayTitle" after the article title

Currently the "After Title" position for fields is misleading as it is actually the exact same place as "Before Display". The only difference is that "After Title" is wrapped into a conditional which checks if "Show Intro" is disabled. Which is confusing as well since the position actually has no ties to the intro text.

### Summary of Changes
This moves the position up after the title and before the info block and removes the conditional.


### Testing Instructions
* Create a field and set it to display "After Title". Assign some value to that field in the article.
* Check in frontend the featured view, blog view and article view.

### Expected result
Field appears after the title


### Actual result
Field appears (if "Show Intro" is disabled) after the info block, right where the fields from "Before Display" appear as well.


### Documentation Changes Required
Not sure.

#### B/C Note
This is not strictly B/C since the fields (and other plugin content using that event) may appear in a different place. Or even appear now when they didn't before (due to param). However this change is only for the Protostar and Beez template and templates that don't override this views. 3rd party templates with overrides to those views will not be affected at all.
